project.clj

```diff
@@ -44,7 +44,7 @@
                  [puppetlabs/trapperkeeper-scheduler]
                  [puppetlabs/trapperkeeper-status]
                  [puppetlabs/kitchensink]
-                 [puppetlabs/ssl-utils]
+                 [puppetlabs/ssl-utils "0.8.2-SNAPSHOT"]
```

Comment:
When we have a non-SNAPSHOT version ready, do you want to override the value from lein-parent here, or update the parent?  Jon updated a few deps in the parent recently so it might be nice to go ahead and rev it?